source/_components/somfy_mylink.markdown

```diff
@@ -0,0 +1,58 @@
+---
+layout: page
+title: "Somfy MyLink"
+description: "Instructions on how to integrate Somfy MyLink devices with Home Assistant."
+date: 2018-10-20 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tahoma.png
+ha_category: Hub
+ha_release: 0.XX
+ha_iot_class: "Assumed State"
+---
+
+
+The `Somfy MyLink` component platform is used as an interface to a compatible Somfy MyLink hub utilizing the `Synergy` API. It adds covers and scenes from the Somfy MyLink platform.
+
+To use your compatible `Somfy MyLink` devices in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+somfy_mylink:
+  host: 10.1.1.100
+  password: mylink_id
+  cover_options:
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
Have removed and put in another more advanced example, based off of the `nest` documentation: https://www.home-assistant.io/components/nest/